content/blog/wingingit/winging-it-09.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Using New Color Formats
+episode: 9
```

Comment:
I'm okay with this, but I still find it a bit confusing that our website now lists episodes 1, 2, 3, 4, 5, 6, and 9. 🤷 

https://deploy-preview-648--oddleventy.netlify.app/wingingit/

Reply:
We just need to add Winging It Live 7 and 8. :)